src/deps_bin/impl/bin.clj

```diff
@@ -36,6 +48,24 @@
     (io/copy (fs/file jar) bin))
   (fs/chmod "+x" bin-file))
 
+(defn ^:private coerce-platforms [coll]
+  (if-some [platforms (not-empty (->> coll (keep allowed-platforms) (distinct)))]
+    platforms
+    [(system-platform)]))
```

Comment:
This is perhaps too forgiving of bad input? May throw if `coll` is not seqable.

Reply:
I don't think is that bad, I'm ok with that